scripts/cleanup-dev-resources.sh

```
@@ -0,0 +1,102 @@
+    --query "Snapshots[?StartTime<='$CUTOFF_DATE'].SnapshotId" \
+    --output text)
+
+if [ ! -z "$SNAPSHOTS" ]; then
+    echo "🗑️  오래된 스냅샷 정리: $SNAPSHOTS"
+    for snapshot in $SNAPSHOTS; do
+        aws ec2 delete-snapshot --snapshot-id $snapshot || echo "⚠️  스냅샷 $snapshot 정리 실패"
+    done
+fi
+
+# Docker 이미지 정리 (개발 서버에서)
+if [ ! -z "$DEV_HOST" ] && [ ! -z "$DEV_SSH_KEY" ]; then
```

Shellcheck (0.10.0)

[style] Use -n instead of ! -z.

(SC2236)